resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/config.scala

```diff
@@ -129,4 +129,22 @@ package object config {
         "when launching drivers. Default is to accept all offers with sufficient resources.")
       .stringConf
       .createWithDefault("")
+
+  private[spark] val MAX_DRIVERS =
+    ConfigBuilder("spark.mesos.maxDrivers").intConf.createWithDefault(200)
```

Comment:
This is not a part of SPARK-26192.

Reply:
It's part of later additions for SPARK-26082 that SPARK-26192 built on in master. I think it's good practice to use these config objects rather than magic strings, and something we want to keep.

Reply:
Backporting the improvement is not allowed. @mwlon . In addition, it's not a good practice to disguise one PR(SPARK-26082) into another PR (SPARK-26192) silently.

Reply:
Although it's not crazy to backport small enhancements, I'd not backport this change. 3.0 changes most all of the configs to use ConfigBuilder, but I don't necessarily think we want to change just a few of them in 2.4, just out of general consistency.